Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -160,15 +160,20 @@ std::unique_ptr<RenderStyle> TreeResolver::styleForStyleable(const Styleable& st
 
 static void resetStyleForNonRenderedDescendants(Element& current)
 {
-    for (auto& child : childrenOfType<Element>(current)) {
+    auto descendants = descendantsOfType<Element>(current);
+    for (auto it = descendants.begin(); it != descendants.end();) {
+        auto& child = *it;
```

Comment:
Should be named 'descendant' now.

Reply:
Or maybe it would be clearer just to do it-> everywhere in the body of this function instead. Note that, if you do that, you'll need to make sure to call clearChildNeedsStyleRecalc() before you call traverseNext().